README.md

```diff
@@ -1,87 +1,110 @@
 ##  Creator-Kit Top level Build system
 
-Creator-Kit requires multiple repositories for building, which are scattered across two GitHub organizations namely CreatorKit, Creatordev, FlowM2M and Cascoda.
+In order to build the IoT Kit projects, we need to collect source code from multiple repositories. The following process will bring these repositories together into a single build environment. The individual repositories used are from these Github organisations:
 
-### Steps for building CreatorKit projects are as follows :-
+* CreatorKit
+* Creatordev
+* FlowM2M
+* Cascoda
 
-Create a directory to keep project repositories, and run following commands :-
+### Building Creator Kit Projects
 
-    $ mkdir creatorkit
-    $ cd creatorkit
-    $ repo init -u https://github.com/CreatorKit/manifest.git -b dev
-    $ repo sync
+#### 1. Install repo tool
+
+To pull together multiple repos, you will need to install the "repo" tool.
+
+On Ubuntu 16.04:
+
+    $ sudo apt-get install repo
 
-_Note :- repo is a tool which should be present on your system. If not then follow this :-_
+On Ubuntu 14.04:
 
     $ mkdir -p ~/bin
     $ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
     $ chmod a+x ~/bin/repo
 
-Update ~/.bashrc to add repo path in linux path permanently :-
+Update ~/.bashrc to add repo path in linux path permanently:
 
     export PATH=$PATH:~/bin/
 
-After repo sync is complete, required repositories are cloned inside the project directory creatorkit.
+#### 2. Sync with the upstream repositories
+
+Create a new directory to keep project repositories, and run the following commands within the directory:
+
+    $ repo init -u https://github.com/CreatorKit/manifest.git
+    $ repo sync
 
-Enter into build repository and start building :-
+After repo sync is complete, the required repositories are cloned inside the project directory.
+
+Optional: If you wish to build from a tagged release, add a tag name to the repo init command as follows:
+
+     $ repo init -u https://github.com/CreatorKit/manifest.git -b 0.9.5
```

Comment:
The correct way of building from a tagged release is mentioned here - https://github.com/CreatorKit/manifest#contents.

And in my opinion, this section of building tagged release is not required in this doc